styleguide/source/_patterns/01-molecules/06-components/09-related-content-item/_related-content-item.scss

```diff
@@ -0,0 +1,82 @@
+/**
+ * @file related-content-item.scss
+ *
+ * Copyright 2017 Palantir.net, Inc.
+ */
+
+// special layout for this
+.layout.layout_related-content_item {
+  @include grid__unit--cols(12);
+  justify-content: flex-start;
+}
+
+.layout_related-content_item_left {
+  @include breakpoint($bp-small) {
+    @include grid__unit--cols(7.5);
+    padding-right: 15px;
+  }
+
+  @include breakpoint($bp-med) {
+    @include grid__unit--cols(6);
+    padding-right: 15px;
+  }
+}
+
+.layout_related-content_item_right {
+  @include breakpoint($bp-small) {
+    @include grid__unit--cols(4.5);
+    padding-right: 15px;
+  }
+
+  @include breakpoint($bp-med) {
+    @include grid__unit--cols(6);
+    padding-right: 15px;
+  }
+}
+
+// non-layout styles
+.related-content_item {
+  display: block;
+  border-top: 1px solid $gray-8;
+  padding: 30px 0;
+
+
+  @include breakpoint($bp-small) {
+    padding: 25px 0;
+  }
+
+  &:hover {
+    .related-content_item_title {
+      text-decoration: underline;
+    }
+  }
+}
+
+.related-content_item_title {
+  font-family: $font-serif;
+
+  @include breakpoint($bp-small min-width) {
+    margin-bottom: 20px;
+  }
+}
+
+.related-content_item_label {
+  color: $purple;
+  text-transform: uppercase;
+}
+
+.related-content_item_image {
+  margin-top: 20px;
+  width: 100%;
+
+  img {
```

Comment:
@aCyborg if you add `display: block` to the `img` tag here it will remove the whitespace below the image wrap. This will fix the VERY subtle positioning of the play button in the middle. Right now it's just below center because of this. :)